sonargraphintegration.properties

```diff
@@ -1,21 +1,21 @@
 #See also: Sonargraph.properties
 
-category=Additional Metrics
+category=External Analyzers
 description=Provides support for Sonargraph 8+'s architecture governance features, accompanied by metrics about cyclic dependencies and other structural aspects.
 homepageUrl=https://github.com/sonargraph/sonar-sonargraph-integration
 devVersion=
-publicVersions=2.1.7,4.0.0
-archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,3.1.0
+publicVersions=2.1.7,4.1.1
+archivedVersions=1.1.2,1.1.3,1.1.4,2.0.1,2.1.5,3.1.0,4.0.0
 privateVersions=
 
 defaults.mavenGroupId=com.hello2morrow
 defaults.mavenArtifactId=sonar-sonargraph-integration
 
-4.0.0.description=Single execution of plugin, compatibility with SonarQube 7.9.1
-4.0.0.sqVersions=[6.7.6,LATEST]
-4.0.0.date=2019-08-29
-4.0.0.changelogUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/tag/Release-4.0.0_with_releasenotes
-4.0.0.downloadUrl=https://github.com/sonargraph/sonar-sonargraph-integration/releases/download/Release-4.0.0_with_releasenotes/sonar-sonargraph-integration-4.0.0.jar
+4.1.1.description=Compatibility with SonarQube 8.1
+4.1.1.sqVersions=[6.7.6,LATEST]
```

Comment:
At a minimum I need you to bump the start of the range to 7.9; support for 6.7 is official over. Also, the description says 8.1 compatibility; should this range actually start there?

Reply:
The minimum version is now set to 7.9.
This release fixes a problem that was introduced with more restrictive security settings in SQ8, leading to classloading problems (https://community.sonarsource.com/t/access-denied-java-lang-runtimepermission-getclassloader-for-plugin-in-computing-engine-of-sq-8-0-and-sq-8-1/17916). Thus, no, compatibility does not start there, but includes the latest version.